src/components/AppToolbar/AppToolbar.tsx

```diff
@@ -15,6 +16,13 @@ import { UserContext } from 'context/UserContextProvider';
 
 import AccountActionButton from './AccountActionButton';
 
+let headerLogo: string | undefined;
+switch (process.env.REACT_APP_AUTH_PROVIDER) {
```

Comment:
I'm not sure if we should be using `REACT_APP_AUTH_PROVIDER` as the environment variable. Possibly `USER_OFFICE_FRONTEND_THEME`.

Reply:
I did question it at one point but then just moved forward with it! Something with theme in it would be more descriptive.

I think it has to include the [`REACT_APP` prefix](https://create-react-app.dev/docs/adding-custom-environment-variables/), so what about `REACT_APP_FRONTEND_THEME`?